### ISSUE TITLE
Question: How easy is it to switch between Builder and Forge?

### ISSUE THREAD

**Issue by TheOddler** [electron-userland/electron-builder#4733]
I've been trying to decide between using electron-builder and electron-forge, and both seem valid options and I have no clue which one to choose. I found #1193, which is very informative, but doesn't really help choosing.

So I thought I'd just pick at random, and later try the other one out. With that in mind I was wondering, how easy is it to later change my mind? Is it perhaps easier to switch from Builder to Forge? Or from Forge to Builder?

Note: I have no experience at all with Electron, so you can assume I know nothing (only googled information at the moment).

**Comment by develar** [comment]
It depends on your needs. Some users needs advanced / regular / portable installer for Windows. In this case only electron-builder provides solution using NSIS.

> how easy is it to later change my mind

Well, it will be easier to switch from electron-forge to electron-builder because electron-builder provides more advanced features than electron-forge. But if you will use advanced features, you will also better understand your needs and requirements. And for most applications such advanced features are not required.

In short, don't pay much attention to this question. Peek whats looks better for you and just start to use it. Later, you can quite easily to switch.

In an ideal world both tools should be non-existent. Since you want to develop your application and do not waste time for boring things like distribution (thanks Ubuntu for snap store, that closes "how to intstall" / "how to update" questions in a very efficient and time-saving manner).

**Comment by kallisti5** [comment]
Keep in mind that electron-builder **STILL** doesn't support Flatpak while electron-forge does.  Flatpak is getting big on pretty much "every distro except Ubuntu"
